docs/getting_started/tasks_and_workflows.md

```diff
@@ -295,8 +295,19 @@ standard_scale_launch_plan = LaunchPlan.get_or_create(
 You can run a `LaunchPlan` locally. It will use the `default_inputs` dictionary
 whenever it's invoked:
 
-```{code-cell} ipython3
-standard_scale_launch_plan()
+Add the following line to your code:
+
+```python
+print(standard_scale_launch_plan())
+```
+Then do:
+
+```bash
+pyflyte run <your-code.py>
+```
+Example output:
+```bash
+[-0.7071067811865475, 0.0, 0.7071067811865475]
```

Comment:
May I know why you've replaced this code block? @davidmirror-ops

Reply:
The way it's rendered doesn't clarify that this should be run with pyflyte instead of the Python interpreter. If the latter, it will result in an error. Maybe there's a better way to clarify it, this is only a proposed and not elegant approach

Reply:
This isn't really trying to explain `pyflyte run` though, it's showing that you can run `standard_scale_launch_plan()` in a python repl.

Reply:
Thank you. I reverted the changes. While I still can't execute the launchplan in the repl, probably I'm missing basic knowledge that our audience already has.

Reply:
nvm, I wasn't using the full wf code in the reply. It works. thanks for your patience :)